app/background/reducers/cosmeticFilterReducer.ts

```diff
@@ -29,6 +29,7 @@ const focusedWindowChanged = (state: State, windowId: number): State => {
       requestShieldPanelData(shieldsPanelState.getActiveTabId(state))
     } else {
       console.warn('no tab id so cannot request shield data from window focus change!')
+      return state
```

Comment:
I don't think you need the extra `return state` here. The time to use that pattern is if you return early because of some condition, e.g.

```js
if (condition) {
 return state
}
...continue processing...
```

Reply:
done